orangecontrib/spectroscopy/widgets/owaverage.py

```diff
@@ -67,10 +67,35 @@ def set_data(self, dataset):
         self.commit()
 
     @staticmethod
-    def average_table(table):
+    def average_table(table, other_vars="None"):
```

Comment:
Wouldn't it be enough to just do "Mean" and not complicate with options. This one keeps the most information. What do you think?

Reply:
Yes, great suggestion. It's easy to get carried away...